src/Widgets/MainView.vala

```diff
@@ -31,8 +31,8 @@ public class SysMonitor.Widgets.MainView : Gtk.Grid {
         attach (ram_value_label,    2, 1, 1, 1);
         attach (swap_label,         1, 2, 1, 1);
         attach (swap_value_label,   2, 2, 1, 1);
-        attach (uptime_label,       1, 3, 1, 1);
-        attach (uptime_value_label, 2, 3, 1, 1);
```

Comment:
I am not sure why did you remove the uptime information? Just add the network info above the uptime row.

Reply:
Corrected, I also do not know why I did this.